configs/orchestrator/cluster-install-configs/oci/oci.lokocfg

```diff
@@ -0,0 +1,117 @@
+variable "cluster_name" {}
+variable "route53_zone" {}
+variable "route53_zone_id" {}
+variable "region" {}
+variable "ssh_pubkey" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "tenancy_id" {}
+variable "compartment_id" {}
+variable "ad_number" {}
+variable "benchmark_worker_count" {}
+variable "benchmark_instance_image_id" {}
+variable "benchmark_instance_type" {}
+variable "benchmark_worker_cpus" {}
+variable "benchmark_worker_memory" {}
+
+variable "amd64_vm_image_id" {
```

Comment:
Why does this variable have the arch and type embedded in the variable? The others don't and that's rather confusing...

Reply:
All the embedded / hardcoded values are something we don't allow user to modify, we have come to these values with some deliberation.

The other values which don't have defaults are expected to be provided by user.

Reply:
Sorry, I don't understand the answer. My question is specifically about this variable called `amd64_vm_image_id`. If we have this variable, we would need to have an `amd64_baremetal_image_id`, an `arm64_vm_image_id` and an `arm64_baremetal_image_id`. Is this what we want?

Reply:
This image_id is used always with the "generic_*" variables. I'm not sure why those are "generic_*" and this one isn't. Also, those values don't seem particularly generic. I think maybe if we called all of these variables "base" instead of "generic" it would be less confusing: `base_image_id`, `base_machine_type`, `base_machine_cpus`, `base_machine_memory`.

Reply:
We decided early on that the BCs will run in hybrid mode. Where the non-benchmarking workloads like prometheus, storage, etc. will run on the AMD64 worker pool. I called this worker pool `generic`. The configuration, in terms of machine type, of both controllers and `generic` workerpool is same so I abstracted that into `amd64_vm_image_id` and the likes.

`amd64_vm_image_id` is only used with the control plane and the worker pool `generic`.

We don't need the likes of `amd64_baremetal_image_id` because we have agreed earlier that the controllers and `generic` worker pool looks same no matter what the benchmarking worker pool looks like. This is the same approach taken in AWS as well.

Sure we call it whatever makes it less ambiguous. The idea I want to get across is that this is common / generic between the the controller and generic worker pool.

Reply:
Yep changing it to have a prefix `base_`.